homeassistant/components/media_player/webostv.py

```diff
@@ -50,10 +61,11 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     if host in _CONFIGURING:
         return
 
-    setup_tv(host, hass, add_devices)
+    customize = config.get(CONF_CUSTOMIZE, None)
```

Comment:
You're defaulting it to `None` which means that if people do not have a customize config entry, the platform will crash. You need to default it to an empty dict `{}`